src/pages/iou/steps/MoneyRequestConfirmPage.js

```diff
@@ -74,6 +75,7 @@ function MoneyRequestConfirmPage(props) {
             }),
         [props.iou.participants, props.personalDetails],
     );
+    const isManualRequestDM = props.selectedTab === CONST.TAB.MANUAL && participants.length < 2;
```

Comment:
```suggestion
    const isManualRequestDM = props.selectedTab === CONST.TAB.MANUAL && iouType.current === CONST.IOU.MONEY_REQUEST_TYPE.REQUEST;
```

We can still split a bill with one person. iouType check is safer

Reply:
Good point! Thanks for the suggestion!